mqsf/exceptions.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2023 SUSE LLC.  All rights reserved.
+#
+# This file is part of mqsf.
```

Comment:
Needs the boilerplate license text. This comment is not sufficient

Reply:
Yep, wasn't sure exactly what license to use so left the boilerplate out of the modules for now. In case we decide to change from Apache 2.0.